### DISCUSSION TITLE
Can I use GH pages as my portfolio?

### DISCUSSION THREAD

**Discussion by EliseyMartynov** [community/community#22795] [Other Feature Feedback, Questions, & Ideas]
<p>Won’t it be commercial use if I will use github pages as portfolio to apply? Or prohibited commercial purposes in terms mean e.g. e-commerce web-site?</p>

**Comment by thomasjo** [comment, marked as answer]
<p>Using it for your portfolio is perfectly fine and not a violation of the terms of service. You can find a summary of the prohibited use cases on the <a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages#prohibited-uses" rel="nofollow noopener">About GitHub Pages</a> help article:</p>
<blockquote>
<p>GitHub Pages is not intended for or allowed to be used as a free web hosting service to run your online business, e-commerce site, or any other website that is primarily directed at either facilitating commercial transactions or providing commercial software as a service (SaaS).</p>
<p>Additionally, GitHub Pages sites must refrain from:</p>
<ul>
<li>Content or activity that is illegal or otherwise prohibited by our <a href="https://docs.github.com/en/articles/github-terms-of-service" rel="nofollow noopener">Terms of Service</a> or <a href="https://docs.github.com/en/articles/github-community-guidelines" rel="nofollow noopener">Community Guidelines</a>
</li>
<li>Violent or threatening content or activity</li>
<li>Excessive automated bulk activity (for example, spamming)</li>
<li>Activity that compromises GitHub users or GitHub services</li>
<li>Get-rich-quick schemes</li>
<li>Sexually obscene content</li>
<li>Content that misrepresents your identity or site purpose</li>
</ul>
</blockquote>
<p><em>(Copied from <a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages" rel="nofollow noopener">https://docs.github.com/en/github/working-with-github-pages/about-github-pages</a> on 2020-08-24 14:49 UTC)</em></p>
<hr>
<p>Additional reference: <a href="https://github.community/t/can-i-use-github-pages-to-host-html5-games-that-will-be-used-in-social-networking-service/1268/2" class="inline-onebox">Can I use GitHub pages to host HTML5 games that will be used in social networking service?</a></p>

**Comment by rachael-fox-ctc** [comment]
<p>To build on this question, what about portfolios for an organisation or company? To showcase clients and give information about the services but are not actually using the website to sell a product - is this within the pages guidelines?</p>

**Comment by thomasjo** [comment]
<p>Hey <a class="mention" href="/rachael-fox-ctc">@rachael-fox-ctc</a>,</p>
<p>I am not a lawyer (IANAL) so take that into account, but as far as I can tell using GitHub Pages to promote your business is acceptable. However, you cannot sell anything (products, services, or other goods, whether digital or physical) via GitHub Pages.</p>
<p>If you are very careful about how you construct your site, you should be OK. But you have to careful because you’re likely going to end up in some gray area of the terms of service. In the end, if you’re unsure if you’re violating the terms, pay for a commercial hosting solution somewhere else, e.g. DigitalOcean, Netlify, etc. Most of these providers offer hosting services starting at around $5 USD per month.</p>
